docs/pagination.md

```diff
@@ -51,11 +51,11 @@ For more information on pagination using Laravel, check out [Laravel's comprehen
 
 ## Resetting the page
 
-When sorting or filtering results, it is common to want to reset the page number back to `1`.
```

Comment:
I'm 50/50 on this change, I think "to want to" reads better, but not overly against your change either.

Reply:
Need to change this back to "to want to".